internal/broker/bindings/bindings_manager.go

```diff
@@ -40,16 +41,16 @@ func NewTokenRequestsBindingsManager(clientProvider ClientProvider, kubeconfigPr
 	}
 }
 
-func (c *TokenRequestsBindingsManager) Create(ctx context.Context, runtimeID, bindingID string) (string, error) {
-	clientset, err := c.clientProvider.K8sClientSetForRuntimeID(runtimeID)
+func (c *TokenRequestsBindingsManager) Create(ctx context.Context, instance *internal.Instance, bindingID string) (string, error) {
```

Comment:
Why we decided to pass entire Instance object when still using only `runtimeID`?

Reply:
I wanted to keep the same interface for both `GardenerBindingManager` and `TokenRequestsBindingsManager`. To do so I need to pass `internal.Instance` because `GardenerBindingManager` uses its shoot field and `TokenRequestsBindingsManager` its `runtimeId` field.